src/runtime_src/xdp/profile/plugin/device_offload/opencl/opencl_device_offload_plugin.cpp

```diff
@@ -338,11 +338,15 @@ namespace xdp {
 
         // Construct the argument list of each port
         std::string arguments = "" ;
-        for (auto arg : matchingCU->get_symbol()->arguments)
+        for (const auto& arg : matchingCU->get_args())
         {
-          if ((arg.address_qualifier != 1 && arg.address_qualifier != 4) ||
-              arg.atype != xocl::xclbin::symbol::arg::argtype::indexed)
-            continue ;
```

Comment:
@hackwa This is repeated few times below but does not look right.  I check with @jvillarre and he wasn't sure.
Skipping non-indexed args is fine. But skipping any args where address_qualifier is different from '1' (global) and '4' (stream) doesn't make sense to me?  I think the intend is to skip non-memory banked arguments, which are actually '1' (global) and '2' (constant) ?

The changes in this PR preserves the old behavior, but please review and confirm.

Reply:
hey Soren this looks good to me.